wa/workloads/gfxbench/__init__.py

```diff
@@ -51,7 +51,7 @@ def update_output(self, context):
         super(Gfxbench, self).update_output(context)
         expected_results = len(self.regex_matches)
         logcat_file = context.get_artifact_path('logcat')
-        with open(logcat_file, errors='replace') as fh:
+        with open(logcat_file, encoding="utf8", errors='ignore') as fh:
```

Comment:
Was this still causing issues with the xpereia device? Should this be updated across other workloads?

Reply:
Reverted this line, I had authored this commit before the logcat parsing change and made mistake in the rebase, sorry about that.